src/models/msg.js

```diff
@@ -46,6 +46,9 @@ class Msg {
 			this.type !== Msg.Type.ERROR &&
 			this.type !== Msg.Type.BANLIST &&
 			this.type !== Msg.Type.IGNORELIST &&
+			this.type !== "channel_list" &&
```

Comment:
Why are these not in the enum? They probably should be.

Reply:
That's what I meant, these types are all gone in Vue which is why I didn't bother adding them to the enum.

Reply:
Oh I missed that in your description. Ignore me. Alrighty.